### PR TITLE
Erkan

### PR DESCRIPTION
**Pull request by merkanb**
hi friends pls share your opinilons. thanks...

### CODE REVIEW

**Comment by seyfettin-hasoz** [comment]
thanks. It seems very good.

**Comment by Cihat46** [comment]
I have checked it it looks good to me

**Comment by merkanb** [comment]
thanks see u guys

File: src/Erkan.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class Erkan {
+    public static void main(String[] args) {
+        System.out.println("hi..");
```
Comment (Cihat46) [inline src/Erkan.java:3]: Dont forget to add Testting

**Review by Cihat46 — CHANGES_REQUESTED** [review]

**Review by seyfettin-hasoz — APPROVED** [review]
thank you

**Review by Cihat46 — APPROVED** [review]

**Pull request merged**